data/events.yml

```diff
@@ -13,6 +13,9 @@ webhook_timeout:
   description: Receive a notification when your webhook times out.
   payload:
     run: [id, description, environment]
+integration_test:
+  title: Integration Test
+  description: Receive a notification to make sure your integration works.
```

Comment:
Instead of adding this as an event along with all the other real ones, it is probably better to make an exception for this event type as it's definitely atypical. This yml file is mainly for the events that are expected to show up in the actual app.